jupyter/d2l-java/chapter_convolutional-modern/vgg.ipynb

```diff
@@ -0,0 +1,530 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%loadFromPOM\n",
+    "<dependency>\n",
+    "    <groupId>tech.tablesaw</groupId>\n",
+    "    <artifactId>tablesaw-jsplot</artifactId>\n",
+    "    <version>0.30.4</version>\n",
+    "</dependency>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load ../utils/plot-utils.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import java.nio.file.*;\n",
+    "import ai.djl.*;\n",
+    "import ai.djl.Model;\n",
+    "import ai.djl.basicdataset.FashionMnist;\n",
+    "import ai.djl.metric.Metrics;\n",
+    "import ai.djl.ndarray.NDArray;\n",
+    "import ai.djl.ndarray.NDManager;\n",
+    "import ai.djl.ndarray.types.DataType;\n",
+    "import ai.djl.ndarray.types.Shape;\n",
+    "import ai.djl.nn.Activation;\n",
+    "import ai.djl.nn.Blocks;\n",
+    "import ai.djl.nn.SequentialBlock;\n",
+    "import ai.djl.nn.convolutional.Conv2D;\n",
+    "import ai.djl.nn.core.Linear;\n",
+    "import ai.djl.nn.norm.Dropout;\n",
+    "import ai.djl.nn.pooling.Pool;\n",
+    "import ai.djl.training.DefaultTrainingConfig;\n",
+    "import ai.djl.training.EasyTrain;\n",
+    "import ai.djl.training.Trainer;\n",
+    "import ai.djl.training.dataset.Dataset;\n",
+    "import ai.djl.training.dataset.ArrayDataset;\n",
+    "import ai.djl.training.evaluator.Accuracy;\n",
+    "import ai.djl.training.listener.TrainingListener;\n",
+    "import ai.djl.training.loss.Loss;\n",
+    "import ai.djl.training.optimizer.Optimizer;\n",
+    "import ai.djl.training.optimizer.learningrate.LearningRateTracker;\n",
+    "import ai.djl.modality.cv.transform.Resize;\n",
+    "import ai.djl.modality.cv.transform.ToTensor;\n",
+    "import ai.djl.translate.Pipeline;\n",
+    "import tech.tablesaw.api.*;\n",
+    "import tech.tablesaw.plotly.api.*;\n",
+    "import tech.tablesaw.plotly.components.*;\n",
+    "import tech.tablesaw.plotly.Plot;\n",
+    "import tech.tablesaw.plotly.components.Figure;\n",
+    "import org.apache.commons.lang3.ArrayUtils;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "attributes": {
+     "classes": [],
+     "id": "",
+     "n": "1"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "public void vggBlock(SequentialBlock block, int numConvs, int numChannels) {\n",
+    "\n",
+    "        SequentialBlock tempBlock = new SequentialBlock();\n",
+    "        for (int i = 0; i < numConvs; i++) {\n",
+    "\n",
+    "            tempBlock\n",
+    "                    .add(Conv2D.builder()\n",
+    "                            .setFilters(numChannels)\n",
+    "                            .setKernelSize(new Shape(3, 3))\n",
+    "                            .optStride(new Shape(1, 1))\n",
```

Comment:
Please check if we need to set this stride value. DJL provides default that could match what Gluon did